compyle/cuda.py

```diff
@@ -130,9 +130,9 @@ def match_dtype_to_c_struct(device, name, dtype, context=None):
         field_dtype, offset = dtype_and_offset[:2]
         c_fields.append("  %s %s;" % (dtype_to_ctype(field_dtype), field_name))
 
-    c_decl = "typedef struct {\n%s\n} %s;\n\n" % (
-        "\n".join(c_fields),
-        name)
+    c_decl = "struct %s {\n%s\n};\n\n" % (
```

Comment:
Just curious as to why this has changed?

Reply:
We use a typedef when we want to define the struct in a convenient way so it can be reused.  So I am not sure why you have changed this.

Reply:
I need to overload the assignment operator for the minmax kernel to work on CUDA

Reply:
https://github.com/pypr/compyle/pull/23/files#diff-d715bedbad67974554356a4c207136c5R98